src/core/storage.tf

```
@@ -0,0 +1,40 @@
+resource "azurerm_resource_group" "rg_storage" {
+  name     = format("%s-storage-rg", local.project)
+  location = var.location
+  tags     = var.tags
+}
+
+## Storage account to save blob
+#tfsec:ignore:azure-storage-default-action-deny
+module "selc-blob-storage" {
+  source = "git::https://github.com/pagopa/azurerm.git//storage_account?ref=v1.0.79"
+
+  name                       = replace(format("%s-blobstorage", local.project), "-", "")
+  account_kind               = "StorageV2"
+  account_tier               = "Standard"
+  account_replication_type   = "LRS"
+  access_tier                = "Hot"
+  enable_versioning          = false
```

```suggestion
  enable_versioning          = false
```
enable versioning only for production